fanuc_lrmate200ic5h_moveit_config/launch/move_group.launch

```diff
@@ -5,15 +5,16 @@
   <!-- GDB Debug Option -->
   <arg name="debug" default="false" />
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
-  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+  <arg     if="$(arg debug)" name="launch_prefix"
+	   value="gdb -x $(find fanuc_lrmate200ic5h_moveit_config)/launch/gdb_settings.gdb --ex run --args" />
```

Comment:
Just checked: running `gdb` with a non-existent command file will just result in a warning / notice:
```
/some/path/command_file.gdb: No such file or directory.
```
it will still just start the `gdb` session.

I'm guessing this is really developer convenience, but I can open an issue on the msa tracker if you'd like.